container/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node
```

Comment:
it would be good to be able to use an alpine or slim version of the image ( also part of the official node images )

Reply:
I'll test the alpine version and include.

Reply:
done.